tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -298,8 +298,8 @@
                 TermStoreDetails: {
                     TermGroup: configs.taxonomy.termGroup,
                     TermSetName: configs.taxonomy.practiceGroupTermSetName,
-                    CustomPropertyName: "ContentTypeName",
-                    DocumentTemplatesName: "DocumentTemplates"
+                    CustomPropertyName: configs.taxonomy.subAreaOfLawCustomContentTypeProperty,

```

Comment:
Removed the hard coding of  ContentTypeName and DocumentTemplates and these are read from config.js